src/merkle/simple_smt/mod.rs

```diff
@@ -222,34 +239,30 @@ impl Store {
         (store, root)
     }
 
-    fn get_empty_node(&self, depth: usize) -> BranchNode {
-        let digest = self.empty_hashes[depth];
-        BranchNode {
-            left: digest,
-            right: digest,
-        }
+    fn replace_empty_hashes(&mut self, hashes: Vec<RpoDigest>) {
+        self.empty_hashes = hashes;
     }
```

Comment:
nit: maybe we should rename this into `replace_emtpy_subtrees()`?

Reply:
Done!